ci/recipe/meta.yaml

```diff
@@ -8,18 +8,20 @@
 # - Set the `number` in the `build` section to the appropriate build number
 # - Remove this comment from the beginning of the file
 
+{% set version = "1.4.0" %}
+
```

Comment:
Can you remove this? We edit this file via bash for the packaging build, and I think we add this automatically.